best403unlocker

```diff
@@ -8,11 +8,49 @@ if [ "$EUID" -ne 0 ]; then
   exit 1
 fi
 }
+install_required_packages(){
+    if [[ -f /etc/os-release  ]]; then
+        source /etc/os-release
+
+        case $ID in
```

Comment:
I think using the ID_LIKE variable is more appropriate because it's more general than ID and can be controlled more easily.

Reply:
Some operating systems do not have ID_LIKE in /etc/os-release.

Reply:
It's OK 
You can use ID